Meta/check-flatpak.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2025, The Ladybird contributors
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+import glob
+import json
+import sys
+
+from enum import Enum
+
+VCPKG = "vcpkg.json"
+VCPKG_OVERLAYS_PORTS = "Meta/CMake/vcpkg/overlay-ports/*"
+FLATPAK_MANIFEST = "Meta/CMake/flatpak/org.ladybird.Ladybird.json"
+SELF = "Ladybird"
+
+# List of build tools that are not provided by the Flatpak SDK and therefor in the manifest
+# For a vcpkg build these are installed on the host system
+flatpak_build_tools = [
+    "gn",
+]
+
+# List of libraries that are dependencies of dependencies
+# These are in the manifest to fetch before the build but not explicitely versioned in vcpkg
+flatpak_deps_of_deps = [
+    "brotli",
+    "dav1d",
+    "highway",
+    "libdwarf",
+    "libxml2",
+    "libyuv",
+    "vulkan-memory-allocator",
+    "zstd",
+]
+
+# List of libraries that are in the Flatpak runtime and therefor not in the manifest
+# See: https://docs.flatpak.org/en/latest/available-runtimes.html#check-software-available-in-runtimes
+flatpak_runtime_libs = [
+    "curl",
+    "dbus",
+    "ffmpeg",
+    "fontconfig",
+    "harfbuzz",
+    "libjpeg-turbo",
+    "libproxy",
+    "openssl",
+    "qtbase",
+    "qtmultimedia",
+    "sqlite3",
+    "zlib",
```

Comment:
Does this mean that we're linking against whatever the kde runtime gives us for these libraries in terms of their versions? These are all pretty critical, wouldn't it be safer to build against the exact same versions as those in vcpkg.json?